src/Microsoft.DotNet.Build.Tasks/Targets/packageresolve.targets

```diff
@@ -87,7 +87,7 @@
                                OmitTransitiveCompileReferences="$(OmitTransitiveCompileReferences)">
       <Output TaskParameter="ResolvedAnalyzers" ItemName="Analyzer" />
       <Output TaskParameter="ResolvedReferences" ItemName="Reference" />
-      <Output TaskParameter="ResolvedCopyLocalItems" ItemName="None" />
+      <Output TaskParameter="ResolvedCopyLocalItems" ItemName="CopyLocalItems" />
```

Comment:
Please merge in my changes.  If you need to get just nuget copy local assets the right way to do that is 
`<Foo Include="@(ReferenceCopyLocalPaths)" Condition="'%(ReferenceCopyLocalPaths.NugetPackageId)' != ''
" />`

As was mentioned in the other comment you might want to consider more than just the packages.  Consider either all of ReferenceCopyLocalPaths (defined after ResolveReferences target closure runs), or more.  To get an idea of everything that might be relevant have a look at the CopyFilesToOutputDirectory target in ms.common.targets and all the targets it depends on.